choccy_chip/src/emulator/sound.rs

```diff
@@ -0,0 +1,56 @@
+use std::time::Duration;
+use std::fmt;
+use rodio::{OutputStreamHandle, OutputStream, Sink};
+use rodio::source::{SineWave, Source};
+
+/// The `Sound` struct is used to play audio in the CHIP-8 emulator.
+pub struct Sound {
+    sink: Sink,
+    stream_handle: OutputStreamHandle,
+    stream: OutputStream,
+}
+
+impl Sound {
+    #[must_use]
+    #[allow(clippy::new_without_default)]
+    /// Creates a new instance of the Sound struct.
+    ///
+    /// # Panics
+    ///
+    /// This function panics if it fails to get the default output stream or create the sink.
+    pub fn new() -> Self {
+        let (stream, stream_handle) = OutputStream::try_default().expect("Failed to get default output stream");
+        let sink = Sink::try_new(&stream_handle).expect("Failed to create sink");
+        sink.pause();
+    
+        Self { sink, stream_handle, stream }
+    }
+
+    /// Plays the sound.
+    pub fn play(&self) {
+        let source = SineWave::new(440.0).take_duration(Duration::from_secs_f32(0.25)).amplify(0.20);
```

Comment:
Make sure to write in the documentation what choices you made. What does 440 mean? Is it just a magic number/ constant.